### PR TITLE
flameshot: Add version 0.9.0

### LINKED ISSUE #5631: [Request] Add flameshot
Powerful yet simple to use screenshot software.
https://github.com/flameshot-org/flameshot
https://flameshot.org

### PR DESCRIPTION
**Pull request by MunsterPlop**
* Closes #5631

The portable archive name only has the format`flameshot_portable-$version-win64.zip` since `0.9.0`.
It had the format `flameshot-$version-win64.zip` in older versions, so it is uncertain that the format will remain the same in the future.

### CODE REVIEW

File: bucket/flameshot.json [hunk 1]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "0.9.0",
+    "description": "Powerful yet simple to use screenshot software.",
```
Comment (Ash258) [inline bucket/flameshot.json:3]: It is not sentence

```suggestion
    "description": "Powerful yet simple to use screenshot software",
```

File: bucket/flameshot.json [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+    "homepage": "https://flameshot.org/",
+    "license": "GPL-3.0",
```
Comment (Ash258) [inline bucket/flameshot.json:5]: Invalid spdx identifier

File: bucket/flameshot.json [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+        "64bit": {
+            "url": "https://github.com/flameshot-org/flameshot/releases/download/v0.9.0/flameshot_portable-0.9.0-win64.zip",
+            "hash": "4e1844c6198f39a1dfbceefc3eb8f373a6d63f7d4dfd97e9a058549744b79ff5",
+            "extract_dir": "flameshot-0.9.0-win64"
+        }
+    },
+    "bin": "bin/flameshot.exe",
+    "shortcuts": [
+        [
+            "bin/flameshot.exe",
```
Comment (Ash258) [inline bucket/flameshot.json:16]: Every manifest in official buckets is using native windows backslashes

**Review by Ash258 — CHANGES_REQUESTED** [review]

File: bucket/flameshot.json (hunk 3 above)
Comment (MunsterPlop) [reply bucket/flameshot.json]: My bad, that's what happens when switching too much between platforms.

File: bucket/flameshot.json [hunk 4]
```diff
@@ -0,0 +1,31 @@
+{
+    "version": "0.9.0",
+    "description": "Powerful yet simple to use screenshot software",
+    "homepage": "https://flameshot.org/",
+    "license": "GPL-3.0-only",
+    "architecture": {
```
Comment (Ash258) [inline bucket/flameshot.json:6]:
```suggestion
    "suggest": {
        "vcredist2019": "vcredist2019"
    },
    "architecture": {
```

File: bucket/flameshot.json [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+        "64bit": {
+            "url": "https://github.com/flameshot-org/flameshot/releases/download/v0.9.0/flameshot_portable-0.9.0-win64.zip",
+            "hash": "4e1844c6198f39a1dfbceefc3eb8f373a6d63f7d4dfd97e9a058549744b79ff5",
+            "extract_dir": "flameshot-0.9.0-win64"
+        }
+    },
```
Comment (Ash258) [inline bucket/flameshot.json:12]:
```suggestion
    },
    "pre_install": "Remove-Item \"$dir\\vc_redist*.exe\"",
```

**Review by Ash258 — CHANGES_REQUESTED** [review]

**Review by Ash258 — APPROVED** [review]

**Pull request merged**